libraries/AP_RCProtocol/AP_RCProtocol_CRSF.h

```diff
@@ -252,29 +252,50 @@ class AP_RCProtocol_CRSF : public AP_RCProtocol_Backend {
         PROTOCOL_ELRS
     };
 
+    // Source for ELRS RF modes: https://www.expresslrs.org/info/signal-health/#rf-mode-indexes-rfmd
     enum RFMode {
         CRSF_RF_MODE_4HZ = 0,
         CRSF_RF_MODE_50HZ,
         CRSF_RF_MODE_150HZ,
         CRSF_RF_MODE_250HZ,
-        ELRS_RF_MODE_4HZ,
```

Comment:
You need both here, just assign them the same numeric value

Reply:
Ok, thanks for the heads-up. Added both with the same numeric value.